app/assets/javascripts/statuses.js

```diff
@@ -0,0 +1,62 @@
+$(function() {
+  $(".proxy-version").each(function(index, item) {
+    var item = $(item);
+    var url = item.data('url');
+    $.ajax({
+      type: 'get',
+      url:  url,
+      success: function(response) {
+        setItemMessage(item, response);
+      }
+    })
+  });
+
+  $(".proxy-status, .compute-status").each(function(index, item) {
+    var item = $(item);
+    var url = item.data('url');
+    $.ajax({
+      type: 'post',
+      url:  url,
+      success: function(response) {
+        setItemStatus(item, response);
+      }
+    })
+  });
+
+  $(".proxy-show-status").each(function(index, item) {
+      var item = $(item);
+      var url = item.data('url');
+      $.ajax({
+        type: 'post',
+        url:  url,
+        success: function(response) {
+          setItemStatus(item, response);
+        }
+      })
+    });
+});
+
+function setItemStatus(item, response) {
+  if(response.status == "OK") {
+    item.attr('title', __('Active'));
+    item.addClass('text-success');
+    item.html('<span class="glyphicon glyphicon-ok-sign" />');
```

Comment:
These icons are different to the host#index page, it should be the same I think.